File: app/src/main/java/com/example/huber/live/entity/LiveEntry.java

package com.example.huber.live.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LiveEntry {

    @SerializedName("data")
    @Expose
    private Data data;

    public Data getData() {
        return data;
    }

    public void setData(Data data) {
        this.data = data;
    }
}

File: app/src/main/java/com/example/huber/entity/LiveEntry.kt
package com.example.huber.entity

import com.google.android.gms.maps.model.LatLng

data class LiveEntry (val direction: Char, val name:String, val richtungsId: Int, val towards: String, val type: String, val latLng: LatLng, val rbl: Int?, val departures: ArrayList<Departure>) {


}

